clustering/web/infinispan/src/main/java/org/wildfly/clustering/web/infinispan/session/fine/FineImmutableSessionAttributes.java

```diff
@@ -55,10 +57,19 @@ public FineImmutableSessionAttributes(String id, Cache<SessionAttributeCacheKey,
     @Override
     public Object getAttribute(String name) {
         SessionAttributeCacheKey key = this.createKey(name);
-        return this.marshaller.read(this.cache.get(key));
+        return this.read(name, this.cache.get(key));
     }
 
     protected SessionAttributeCacheKey createKey(String attribute) {
         return new SessionAttributeCacheKey(this.id, attribute);
     }
+
+    protected Object read(String name, V value) {
+        try {
+            return this.marshaller.read(value);
+        } catch (InvalidSerializedFormException e) {
+            InfinispanWebLogger.ROOT_LOGGER.failedToActivateSessionAttribute(e, this.id, name);
```

Comment:
The exception caught is a checked exception whereas the exception that is rethrown is a runtime exception.  The point of logging here is because we have context to record which isn't available later.
But, your right, I should be supplying the context in the rethrown exception message, not logging it here.  I'll fix that.